.gitignore

```diff
@@ -1,2 +1,7 @@
 *.zip
 keys
+dist/
+node_modules/
+.netlify

```

Comment:
`.netlify` doesn't need to be ignored right? We're just not using it anymore

Reply:
It's different from the folder you had earlier. If we need serverless functions again, we'll run `netlify dev` instead to get those running locally. The `.netlify` folder is created during `netlify dev`.

Reply:
I see. it's a generated file, so we'll ignore it.